### PR TITLE
Fix TODO - bot names

### PR DESCRIPTION
**Pull request by jake-87**
Added GH-style default names to the bots, using a list of animals.

### CODE REVIEW

File: client/necsus.js [hunk 1]
```diff
@@ -219,9 +219,26 @@ let app = new Vue({
...
+                       "Dragonfly", "Duck", "Eagle", "Eel", "Elephant", "Emu", "Falcon", "Ferret", "Finch",
+                       "Fish", "Flamingo", "Flea", "Fly", "Fox", "Frog", "Goat", "Goose", "Gopher", "Gorilla",
+                       "Hamster", "Hare", "Hawk", "Hippopotamus", "Horse", "Hummingbird", "Husky", "Iguana",
+                       "Impala", "Kangaroo", "Lemur", "Leopard", "Lion", "Lizard", "Llama", "Lobster", "Margay",
+                       "Monkey", "Moose", "Mosquito", "Moth", "Mouse", "Mule", "Octopus", "Orca", "Ostrich",
+                       "Otter", "Owl", "Ox", "Oyster", "Panda", "Parrot", "Peacock", "Pelican", "Penguin",
+                       "Perch", "Pheasant", "Pig", "Pigeon", "Porcupine", "Quagga", "Rabbit", "Raccoon", "Rat",
+                       "Rattlesnake", "Rooster", "Seal", "Sheep", "Skunk", "Sloth", "Snail", "Snake", "Spider",
+                       "Tiger", "Whale", "Wolf", "Wombat", "Zebra"]
+      const shuffled = animals.sort(() => 0.5 - Math.random());
```
Comment (joelgibson) [inline client/necsus.js]: Sorting using a random comparator is not quite the same thing as shuffling, and doesn't make much sense (the sorting algorithm could ask if `x <= y` and get `true`, and later ask the same question and get `false`). How you can get a random shuffle via sorting is by generating a random number for each element of the list, hence turning every item into a pair of `(data, random number)`, and sorting by the random number, like in this answer: <https://stackoverflow.com/a/46545530>

**Review by joelgibson — COMMENTED** [review]
Hi Jake, thanks for the PR! If you can change your shuffling algorithm to something which is genuinely a shuffle (or some other way of choosing three items at random without replacement), then I'll happily merge it.

**Comment by jake-87** [comment]
Should be good to go now. Apologies for mostly copying directly from SO, but it seemed like the most reasonable way to do it.

**Comment by jake-87** [comment]
Enjoy NCSS 2024 ;)

**Pull request merged**